src/rdkafka_msgset_writer.c

```diff
@@ -831,15 +831,15 @@ rd_kafka_msgset_writer_write_msgq (rd_kafka_msgset_writer_t *msetw,
 
         /* Idempotent Producer: Acquire BaseSequence from first message */
         if (rd_kafka_pid_valid(rktp->rktp_eos.pid)) {
-                /* Our sequence counter is 64-bits, but the
+                /* Our msgid counter is 64-bits, but the
```

Comment:
Why not just use a signed 64 bit integer instead?

Reply:
Right so is this future proofing for when they change the msg format in Kafka or is it necessary to handle wraps in the client.